### ISSUE TITLE
Catching lupe when opening and/or importing different formats

### ISSUE THREAD

**Issue by waxfingers** [aferrero2707/PhotoFlow#126]
OSX, latest build. Photoflow enters catching loop when opening and/or adding a layer with 32bit tifs, 16 and 8 bits dpx and portable bit maps also 8 bits bmp and targa files.

**Comment by aferrero2707** [comment]
Could you post a screenshot of the layers that are active when caching?

**Comment by waxfingers** [comment]
I don't think screenshot is necessary; opening (drag and dropping) the above mentioned files into PF loads raw loader and raw developer modules which then enter caching loop, same IF adding them as a layer image. I might just be forcing the program, maybe those formats are not supported  (?) - as they do not appear from normal opening dialog, just when dragging (showing all files) into it. To my surprise today PF loaded the 32 bit tiff no problem.
Anyway from an user standpoint we are left wondering if there is a bug or the formats are not supported or _what did I do wrong?_ Hey you have a great day ,-)

**Comment by aferrero2707** [comment]
You are right, all those formats except 32 bits TIFFs are not supported... TIFF should always be loaded as normal images, not RAW files.

I agree that I should add a warning dialog telling that the input format is not supported. I'll keep this issue opened until this is added.

Thanks, and a great day to you as well!
